spec/components/dossiers/batch_operation_component_spec.rb

```diff
@@ -22,11 +29,25 @@
   subject { render_inline(component).to_html }
   context 'statut suivis' do
     let(:statut) { 'suivis' }
+
     it { is_expected.to have_button('Passer les dossiers en instruction', disabled: true) }
     it { is_expected.to have_button('Instruire les dossiers', disabled: true) }
     it { is_expected.to have_button('Autres actions multiples', disabled: true) }
     it { is_expected.to have_button('Repasser les dossiers en construction', disabled: true) }
     it { is_expected.to have_button('Ne plus suivre les dossiers', disabled: true) }
+    it { is_expected.to have_button('Demander un avis externe', disabled: true) }
+
+    context 'with expert review disallowed procedure' do
+      before {
+        procedure.update!(allow_expert_review: false)
+      }
+      it { is_expected.to have_button('Passer les dossiers en instruction', disabled: true) }
+      it { is_expected.to have_button('Instruire les dossiers', disabled: true) }
+      it { is_expected.to have_button('Autres actions multiples', disabled: true) }
+      it { is_expected.to have_button('Repasser les dossiers en construction', disabled: true) }
+      it { is_expected.to have_button('Ne plus suivre les dossiers', disabled: true) }
+      it { is_expected.not_to have_button('Demander un avis externe', disabled: true) }
```

Comment:
Remarque générale pour les tests en particulier de controller qui initialisent bcp d'objets en base: est-ce que tu pourrais regrouper toutes les vérifications qui utilisent le même setup dans un seul `it` ? par exemple quelque chose comme ça : 

```ruby
it do
  is_expected.to have_button('Passer les dossiers en instruction', disabled: true)
  is_expected.to have_button('Instruire les dossiers', disabled: true)
  …
end
```

comme ça on fait qu'une seule fois le setup et l'appel au controller pour l'ensemble des tests et on gagne du temps de CI

(je sais que c'est pas le cas juste au dessus, mais c'est l'occasion de faire la transformation aussi pour que le CI aille plus vite ;)